rrdserver.go

```diff
@@ -211,5 +213,9 @@ func main() {
 	http.HandleFunc("/annotations", annotations)
 	http.HandleFunc("/", hello)
 
-	http.ListenAndServe(":"+strconv.Itoa(config.Server.Port), nil)
+	err := http.ListenAndServe(":"+strconv.Itoa(config.Server.Port), nil)
```

Comment:
@gnuheidix Did you mean `http.ListenAndServe(config.Server.IpAddr+":"+strconv.Itoa(config.Server.Port), nil)`?